src/js/navigation.js

```diff
@@ -2,12 +2,54 @@ const navigationToggle = document.querySelector(".menu-btn");
 const navigation = document.querySelector("nav");
 const welcome = document.querySelector(".welcome");
 
-navigationToggle.addEventListener("click", ()  => {
+navigationToggle.addEventListener("click", () => {
   document.body.classList.toggle("nav-is-open");
+  welcome.classList.toggle("welcome-to-right");
   welcome.style.transition = "transform ease-out 250ms";
+  if (welcome.classList.contains("welcome-to-right")) {
+    const welcomeRight = document.querySelector(".welcome-to-right");
```

Comment:
You don't need to select `welcomeRight` again it's the same element as `welcome`

Reply:
welcomeRight je welcome z razredom welcome-to-right, torej welcome samo v tistem trenutku, ko je zamaknjen desno